feconf.py

```diff
@@ -434,7 +434,7 @@ def get_empty_ratings():
 ENABLE_MAINTENANCE_MODE = False
 
 # Whether community dashboard is ready to use for contributors.
-COMMUNITY_DASHBOARD_ENABLED = False
+COMMUNITY_DASHBOARD_ENABLED = True
```

Comment:
Don't enable this yet please. (Right, @DubeySandeep?)

Reply:
Yeah, that's not expected! (We are going to enable it later once we have the images in suggestion functionality.)

Reply:
sorry about that, enabled to test but forgot to disable

Reply:
Done